src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  if (days >= 7) {
+    return (sum -= 50);
+  }
+
+  if (days >= 3) {
+    return (sum -= 20);
```

Comment:
Similarly to the previous comment, replace the literal values with constants for the discount logic here as well.